src/views/product-landing/components/product-landing-blocks/blocks/collection-cards/collection-cards.module.css

```diff
@@ -1,5 +0,0 @@
-.placeholder {
```

Comment:
These `product-landing-blocks` CSS files are unrelated to this PR, I noticed them when elevating the `getInlineContentMaps` utility. They need to be cleaned up; they're not used anywhere.